.github/workflows/github-actions-python.yml

```
@@ -14,7 +14,7 @@ jobs:
       - name: Install dependencies
         run: |
           python -m pip install --upgrade pip
-          pip install flake8 pytest paho-mqtt requests-mock jq pyjwt==2.6.0 bs4 pkce typing_extensions python-dateutil==2.8.2
+          pip install flake8 pytest paho-mqtt requests-mock jq pyjwt==2.6.0 bs4 pkce typing_extensions python-dateutil==2.8.2 msal onedrive-sharepoint-python-sdk
```

```suggestion
          pip install flake8 pytest paho-mqtt requests-mock jq pyjwt==2.6.0 bs4 pkce typing_extensions python-dateutil==2.8.2 msal
```
Die Libary wird nicht verwendet.

---

diese library liefert msdrive welches verwendet wird. Warum die Namen unterschiedlich sind erschließt sich mir allerdings nicht. https://pypi.org/project/onedrive-sharepoint-python-sdk/

---

removed library